snuba/consumer.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, dataset, producer, replacements_topic, metrics=None):
         self.metrics = metrics
         self.__writer = dataset.get_writer({
             'load_balancing': 'in_order',
-            'insert_distributed_sync': True,
+            'insert_distributed_sync': 1,
```

Comment:
I guess to be technically correct (the best kind of correct) this should actually be a string, and not an integer.